package.json

```diff
@@ -1,40 +1,34 @@
 {
-  "name": "auto-sdk-ts",
-  "version": "1.0.0",
-  "description": "",
-  "main": "index.js",
+  "name": "@subspace/auto",
+  "private": true,
   "scripts": {
-    "postinstall": "node prebuild.js",
-    "build": "tsc",
-    "clean": "rm -rf dist",
-    "clean:all": "rm -rf dist && rm -rf node_modules",
-    "start": "tsc && node dist/index.js",
-    "lint": "eslint . --ext .ts",
-    "format": "prettier --write \"src/**/*.ts\" \"tests/**/*.ts\"",
-    "eg:all": "tsc && node dist/examples/genEvmAddrs.js && node dist/examples/genAutoId.js && node dist/examples/genAutoWallet.js && node dist/examples/sss.js && node dist/examples/e2eSeedRecovery.js",
-    "eg:gen-evm-addrs": "tsc && node dist/examples/genEvmAddrs.js",
-    "eg:gen-auto-id": "tsc && node dist/examples/genAutoId.js",
-    "eg:gen-auto-wallet": "tsc && node dist/examples/genAutoWallet.js",
-    "eg:sss": "tsc && node dist/examples/sss.js",
-    "eg:e2e-seed-recovery": "tsc && node dist/examples/e2eSeedRecovery.js"
+    "dev:auto-app": "turbo run dev --continue --filter=@subspace/auto-app...",
+    "dev": "turbo run dev",
+    "build": "turbo run build",
+    "clean": "turbo run clean",
```

Comment:
After using `$ pnpm clean`, I don't find the repo cleaned. 

Before clean:
<img width="214" alt="image" src="https://github.com/subspace/auto-sdk-ts/assets/16472948/ed792d20-1a88-4e74-840f-1f3ee556a2cc">

After clean:
<img width="242" alt="image" src="https://github.com/subspace/auto-sdk-ts/assets/16472948/bcac42ad-1d15-4a29-b691-81cdea667773">

Hence, I think, we need to delete the folders like this:
```json
...
    "clean": "rm -rf dist && rm -rf .turbo",
    "clean:all": "rm -rf dist && rm -rf .turbo && rm -rf node_modules",
...
```

And then add the corresponding instructions in monorepo's README.

Reply:
Fixed now.